app/templates/base-cy.html

```diff
@@ -8,7 +8,7 @@
 
 {%- if ('/cy/start/' in page_url) and (page_url != '/cy/start/') and (timeout != 'true') -%}
     {%- set signout_button_value = {
-            'text': 'PLACEHOLDER WELSH Gadael',
+            'text': 'Gadael',
```

Comment:
I think this whole if block needs removing, as well as where we set the `signoutButton` in the `pageConfig`, we don't have a signout button.